samples/nrf9160/mqtt_simple/src/main.c

```diff
@@ -332,10 +381,39 @@ static void client_init(struct mqtt_client *client)
 
 	/* MQTT transport configuration */
 #if defined(CONFIG_MQTT_LIB_TLS)
+	struct mqtt_sec_config *tls_cfg = &(client->transport).tls.config;
 	client->transport.type = MQTT_TRANSPORT_SECURE;
+
+#if defined(CONFIG_BSD_LIBRARY)
+	static sec_tag_t sec_tag_list[] = { CONFIG_MQTT_TLS_SEC_TAG };
+
+	tls_cfg->session_cache = IS_ENABLED(CONFIG_MQTT_TLS_SESSION_CACHING) ?
+					    TLS_SESSION_CACHE_ENABLED :
+					    TLS_SESSION_CACHE_DISABLED;
+
+	tls_cfg->peer_verify = CONFIG_MQTT_TLS_PEER_VERIFY;
```

Comment:
Isn't it needed to set these parameters for qemu build as well? Right now running the sample for qemu will skip the certificate verification if I understand correctly.

Reply:
Yes, you are correct.